db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
I hate that kind of magic, when: what you see is what you __don't__ get
But maybe it's because I am old and grumpy :)